embark.el

```diff
@@ -3623,13 +3630,14 @@ and leaves the point to the left of it."
   ("SPC" mark)
   ("DEL" delete-region))
 
-;; TODO add more encode actions, see M-x or C-h f encode region
+(autoload 'message-caesar-region "message" nil t)
 (embark-define-keymap embark-encode-map
   "Keymap for Embark region encoding actions."
   :parent nil
-  ("r" rot13-region)
+  ("r" message-caesar-region) ;; default rot13-region
```

Comment:
This one I'm not so sure about: I think the only case of the Caeser cipher people actually use is Δ=13. I think I'd rather not load `message.el` for what is most likely to be `rot13` anyway.

Reply:
Oh, I thought you like the generalization :laughing: 

If I apply a Caesar chipher  to a region I guess I wouldn't bother if message.el is loaded. Is it that heavy?

Reply:
Actually I think this should be fixed in Emacs itself. message-caesar-region should be renamed to rot13-region and removed from message.el. There is no point in having both these commands. But I don't dare to report a bug for this :smile: